<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainfunc</class>
 <widget class="QWidget" name="Mainfunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>777</width>
    <height>539</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>777</width>
    <height>539</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>主菜单</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{

border-image: url(:/background/屏幕截图 2024-06-01 114350.png);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableView" name="contact_table">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>621</width>
      <height>511</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>621</width>
      <height>511</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>161</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#plainTextEdit
{
background-color: rgba(255, 255, 255,0.6);
}
#plainTextEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px dashed #fffac2;
	border-radius: 3px;
	background-color:#fffee2;
	color: #000000;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
#plainTextEdit:focus {
    border: 1px dashed #fffac2;
    border-radius: 3px;
    background-color: #fffee2;
    color: #000000;
    selection-background-color: #298DFF;
    selection-color: #F2F2F2;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="email_pb">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);</string>
    </property>
    <property name="text">
     <string>生日查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HoloLens MDL2 Assets</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>便利贴</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_pb">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(0, 0, 0, 0); /* 最后一个参数是透明度，0表示完全透明 */
    border: none; /* 移除边框 */
    /* 如果您希望文本也透明，可以设置text-color的透明度 */
    color: rgba(255, 255, 255, 0); /* 白色文本，透明度为0 */
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.2); /* 鼠标悬停时轻微的背景色，透明度0.2 */
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 0.4); /* 按下时更明显的背景色，透明度0.4 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_buttom">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);</string>
    </property>
    <property name="text">
     <string>添加联系人</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_le_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>365</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#search_le_8
{
background-color: rgba(255, 255, 255,0.6);
}
#search_le_8:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px dashed #fffac2;
	border-radius: 3px;
	background-color:#fffee2;
	color: #000000;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
#search_le_8:focus {
    border: 1px dashed #fffac2;
    border-radius: 3px;
    background-color: #fffee2;
    color: #000000;
    selection-background-color: #298DFF;
    selection-color: #F2F2F2;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_pb_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;搜索&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource/resource.qrc">
      <normaloff>:/image/R-C.jpg</normaloff>:/image/R-C.jpg</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
